persistence.js

```diff
@@ -100,21 +100,24 @@ class MemoryPersistence {
     }
 
     for (const sub of subs) {
-      const qos = stored.get(sub.topic)
-      const hasQoSGreaterThanZero = (qos !== undefined) && (qos > 0)
+      const storedSub = stored.get(sub.topic)
+      const hasQoSGreaterThanZero = storedSub && (storedSub.qos !== undefined) && (storedSub.qos > 0)
```

Comment:
This can imho be simplified to:
```javascript 
const hasQoSGreaterThanZero = storedSub?.qos > 0
```

Reply:
Yep giving that we have dropped nodejs 12 support now we can safely use it